compiler-rt/include/sanitizer/asan_interface.h

```diff
@@ -333,6 +333,13 @@ void SANITIZER_CDECL __asan_handle_no_return(void);
 /// trace. Returns 1 if successful, 0 if not.
 int SANITIZER_CDECL __asan_update_allocation_context(void *addr);
 
+/// Disables fake stack usage for the current thread.
+/// Temporarily disables use-after-return detection for current thread.
```

Comment:
Please remove the word ""usage" from the comment and function name.
Seems redundunt.